l10n: translations update from Hosted Weblate

weblate opened the pull request:
Translations update from [Hosted Weblate](https://hosted.weblate.org) for [boorusama/app](https://hosted.weblate.org/projects/boorusama/app/).



Current translation status:

![Weblate translation status](https://hosted.weblate.org/widget/boorusama/app/horizontal-auto.svg)

Commit:
l10n: translated using Weblate (Ukrainian)

Currently translated at 100.0% (889 of 889 strings)

Translation: boorusama/app
Translate-URL: https://hosted.weblate.org/projects/boorusama/app/uk/

mhguedes445 committed:
l10n: translated using Weblate (Portuguese (Brazil))

Currently translated at 73.3% (652 of 889 strings)

Translation: boorusama/app
Translate-URL: https://hosted.weblate.org/projects/boorusama/app/pt_BR/

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/khoadng/Boorusama?pullRequest=621) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you all sign our [Contributor License Agreement](https://cla-assistant.io/khoadng/Boorusama?pullRequest=621) before we can accept your contribution.<br/>**0** out of **2** committers have signed the CLA.<br/><br/>:x: mhguedes445<br/>:x: maksim2005UKR<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/khoadng/Boorusama?pullRequest=621) it.</sub>

Commit:
l10n: translated using Weblate (Chinese (Simplified Han script))

Currently translated at 100.0% (889 of 889 strings)

Translation: boorusama/app
Translate-URL: https://hosted.weblate.org/projects/boorusama/app/zh_Hans/

WarHammer-z committed:
l10n: translated using Weblate (Turkish)

Currently translated at 14.8% (132 of 889 strings)

Translation: boorusama/app
Translate-URL: https://hosted.weblate.org/projects/boorusama/app/tr/

Commit:
l10n: translated using Weblate (German)

Currently translated at 70.3% (625 of 889 strings)

Translation: boorusama/app
Translate-URL: https://hosted.weblate.org/projects/boorusama/app/de/

Commit:
l10n: translated using Weblate (German)

Currently translated at 77.0% (685 of 889 strings)

Translation: boorusama/app
Translate-URL: https://hosted.weblate.org/projects/boorusama/app/de/

savvacorgi committed:
l10n: translated using Weblate (Russian)

Currently translated at 53.7% (478 of 889 strings)

Translation: boorusama/app
Translate-URL: https://hosted.weblate.org/projects/boorusama/app/ru/

Commit:
l10n: translated using Weblate (Korean)

Currently translated at 76.8% (683 of 889 strings)

Translation: boorusama/app
Translate-URL: https://hosted.weblate.org/projects/boorusama/app/ko/
